src/pages/sighting/SightingCore.jsx

```diff
@@ -222,22 +209,11 @@ export default function SightingCore({
           </Text>
         )}
       </EntityHeaderNew>
-      {showCommitAlert && (
-        <Alert
-          severity="info"
-          titleId="SIGHTING_COMMIT_TITLE"
-          descriptionId="SIGHTING_COMMIT_DESCRIPTION"
-          action={
-            <Button
-              id="COMMIT"
-              size="small"
-              loading={commitAgsLoading}
-              onClick={() => commitAgs(id)}
-              style={{ marginTop: 8, marginRight: 4 }}
-            />
-          }
-        />
-      )}
+      <CommitBanner
```

Comment:
I didn't know where to put this comment. So... if I have a sighting with no annotations that has completed detection (at least according to the UI), why can't I commit? In other words, is the picture below desired behavior or a bug either with this code or with detection status?
<img width="1552" alt="Screen Shot 2022-01-13 at 12 25 27 PM" src="https://user-images.githubusercontent.com/2775448/149403886-2d9534ba-e82f-4f99-a0bb-259078ce0b65.png">